README.md

```diff
@@ -13,6 +13,7 @@
 | [`model_json`][model_json]                 | Create a basic model class using json serializable                          |
 | [`rest_frog`][rest_frog]                   | Create REST endpoints for a given route in your Dart Frog project           |
 | [`ui_package`][ui_package]                 | Create a new UI package in Flutter containing the design system of your app |
+| [`somnio_starter`][somnio_starter]                 | Create a new UI package in Flutter containing the design system of your app |
```

Comment:
Check format here, although it doesn't matter for the md but if everything is aligned it's better :)